src/request-sender.ts

```diff
@@ -102,6 +103,10 @@ export default class RequestSender {
             defaultOptions.headers['Content-Type'] = 'application/json';
         }
 
+        if (this._options.isControlPanel && defaultOptions.headers) {
+            defaultOptions.headers[this.CONTROL_PANEL_HEADER] = true;
```

Comment:
Just in line the header.

```suggestion
            defaultOptions.headers['X-CONTROL-PANEL-REQUEST'] = true;
```